### ISSUE TITLE
Allow a teacher/admin to see the list of modules with controls

### ISSUE THREAD

**Issue by amellie** [Ali-Jahankah/bugsquashers-edu-app#21]
## Dependency

Depends on #13 

## Story
**As a** Teacher or an Admin
**I Want** to see the list of existing modules
**So That** I can see review these modules and perform actions (create/update/etc)

## Acceptance Criteria
- Show a table in the page. The table should show these columns:
-- Title (of the module)
-- Number of lessons related to the module
-- Action (which contains: Edit and Delete button)
- A Create button should be added to the top of the table. When the button is clicked, it should show the page that has been done in #15 
- Edit and Delete buttons do not do anything when clicked (this will be handled in another task, so please leave it as is)
